lib/modules/manager/npm/extract/index.spec.ts

```diff
@@ -435,7 +435,7 @@ describe('modules/manager/npm/extract/index', () => {
           {
             depName: 'other',
             currentValue: 'latest',
-            skipReason: 'unknown-version',
+            skipReason: 'unspecified-version',
```

Comment:
@rarkins should we really set skip reason here for `latest` current value? 🤔

Reply:
There's an issue open about this: #3945

I'm not sure whether simply unskipping this will work, or further work is done. Suggest leaving it for another PR